Libraries/src/Amazon.Lambda.DynamoDBEvents/DynamoDBEvent.cs

```diff
@@ -21,15 +18,315 @@ public class DynamoDBEvent
+            /// </para>
+            /// 
+            /// <ul>
+            /// <li>
+            /// <para>
+            ///  <c>KEYS_ONLY</c> - only the key attributes of the modified item.
+            /// </para>
+            /// </li>
+            /// 
+            /// <li>
+            /// <para>
+            ///  <c>NEW_IMAGE</c> - the entire item, as it appeared after it was modified.
+            /// </para>
+            /// </li>
+            /// 
+            /// <li>
+            /// <para>
+            ///  <c>OLD_IMAGE</c> - the entire item, as it appeared before it was modified.
+            /// </para>
+            /// </li>
+            /// 
+            /// <li>
+            /// <para>
+            ///  <c>NEW_AND_OLD_IMAGES</c> - both the new and the old item images of the item.
+            /// </para>
+            /// </li>
+            /// </ul>
+            /// </summary>
+            public string StreamViewType { get; set; }
+        }
+
+        /// <summary>
+        /// Contains details about the type of identity that made the request.
+        /// </summary>
+        public class Identity
+        {
+            /// <summary>
+            /// A unique identifier for the entity that made the call. For Time To Live, the principalId
+            /// is "dynamodb.amazonaws.com".
+            /// </summary>
+            public string PrincipalId { get; set; }
+
+            /// <summary>
+            /// The type of the identity. For Time To Live, the type is "Service".
+            /// </summary>
+            public string Type { get; set; }
+        }
+
+        /// <summary>
+        /// Represents the data for an attribute.
+        ///  
+        /// <para>
+        /// Each attribute value is described as a name-value pair. The name is the data type,
+        /// and the value is the data itself.
+        /// </para>
+        ///  
+        /// <para>
+        /// For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.DataTypes">Data
+        /// Types</a> in the <i>Amazon DynamoDB Developer Guide</i>.
+        /// </para>
+        /// </summary>
+        public class AttributeValue
+        {
+            /// <summary>
+            /// <para>
+            /// An attribute of type Binary. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"B": "dGhpcyB0ZXh0IGlzIGJhc2U2NC1lbmNvZGVk"</c>
+            /// </para>
+            /// </summary>
+            public MemoryStream B { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Boolean. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"BOOL": true</c>
+            /// </para>
+            /// </summary>
+            public bool BOOL { get; set; }

```

Comment:
Use the nullable type. Here use `bool?`.

Reply:
Updated in https://github.com/aws/aws-lambda-dotnet/pull/1648/commits/3bd10bc4900e4d21ebba4d47fe9700cd0076a74a#diff-03c240223072abf4e9dea6645203768f84845f84f54ea472b360aeff7d3960cfL229